src/rdf4cpp/rdf/Literal.cpp

```diff
@@ -226,24 +253,127 @@ Literal Literal::to_node_storage(Node::NodeStorage &node_storage) const noexcept
                 return node_storage.find_or_make_id(literal_view);
             },
             [&node_storage, &literal_view](storage::node::view::ValueLiteralBackendView const &any) noexcept {
```

Comment:
```suggestion
            [&node_storage, &literal_view](storage::node::view::ValueLiteralBackendView const &literal_backend) noexcept {
```

Or maybe something else. I find `any` confusing.